### DISCUSSION TITLE
Questions about the Area card

### DISCUSSION THREAD

**Discussion by dave-dube** [dwainscheeren/dwains-lovelace-dashboard#530] [Q&A] [answered]
1- i have 2 light in my kitchen.  One main light, and a second light i use only when required(few times a month).  Actually, because they are both on the same area "kitchen", when clicking on the little light icon on the top right of my kitchen area card, both lights goes on.
is there any way to tell DD what lights to link on this light shortcut ?

2- same kind of question about Temperature showned on the area card.  Actually, an average of all Temperature sensor found is showned on the area card.  Is there a way to tell DD what sensor to consider to show on the area card ?

3- Is there a way to hide an area into DD ?

Thanks

**Comment by dwainscheeren** [comment, marked as answer]
All 3 questions are fixed in 3.1

**Comment by dave-dube** [comment]
Heuuuuu...  Wow really Nice.
 Thanks.
And really nice work!

Le ven. 6 mai 2022 05 h 16, Dwain Scheeren ***@***.***> a
écrit :

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
